README.md

```diff
@@ -409,6 +409,95 @@ crud.replace_object('customers', {
 ...
 ```
 
+### Replace many
+
+```lua
+-- Batch replace tuples
+local result, err = crud.replace_many(space_name, tuples, opts)
+-- Batch replace objects
+local result, err = crud.replace_object_many(space_name, objects, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space to insert/replace an object
+* `tuples` / `objects` (`table`) - array of tuples/objects to insert
+* `opts`:
+  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+  * `fields` (`?table`) - field names for getting only a subset of fields
+  * `stop_on_error` (`?boolean`) - stop on a first error and report errors
+    regarding the failed operation and all not performed ones., default is
+    `false`
+  * `rollback_on_error` (`?boolean`) - any failed operation will lead to
+    rollback on a storage, where the operation is failed, default is
+    `false`
+
+Returns metadata and array contains inserted rows, array of errors
+(one error corresponds to one replicaset for which the error occurred).
+Error object can contain `tuple` field. This field contains the tuple
+for which the error occurred.
+
+Right now CRUD cannot provide batch replace with full consistency.
+CRUD offers batch replace with partial consistency. That means
+that full consistency can be provided only on single replicaset
+using `box` transactions.
+
+**Example:**
+
+```lua
+crud.replace_many('developers', {
+  {1, box.NULL, 'Elizabeth', 'lizaaa'},
+  {2, box.NULL, 'Anastasia', 'iamnewdeveloper'},
+})
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucket_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'login', 'type': 'string'}
+  rows:
+  - [1, 477, 'Elizabeth', 'lizaaa']
+  - [2, 401, 'Anastasia', 'iamnewdeveloper']
+...
+crud.replace_object_many('developers', {
+    {id = 1, name = 'Inga', login = 'mylogin'},
+    {id = 10, name = 'Anastasia', login = 'qwerty'},
+})
+---
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucket_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [1, 477, 'Inga', 'mylogin']
+  - [10, 569, 'Anastasia', 'qwerty']
+
+-- Partial success
+-- Let's say login has unique secondary index
+local res, errs = crud.replace_object_many('developers', {
+    {id = 22, name = 'Alex', login = 'pushkinn'},
+    {id = 3, name = 'Anastasia', login = 'qwerty'},
+    {id = 5, name = 'Sergey', login = 'crudisthebest'},
+})
+---
+res
+- metadata:
+  - {'name': 'id', 'type': 'unsigned'}
+  - {'name': 'bucket_id', 'type': 'unsigned'}
+  - {'name': 'name', 'type': 'string'}
+  - {'name': 'age', 'type': 'number'}
+  rows:
+  - [5, 1172, 'Sergey', 'crudisthebest'],
+  - [22, 655, 'Alex', 'pushkinn'],
+
+#errs              -- 1
```

Comment:
Why don't print it as yaml too?

Reply:
I guess, because it is large. Also, multireturn handling may confuse a reader (tarantool's console always returns a list, because a Lua function can return many results).

Let me show an example (I wrapped the error into the table to imitate `crud.*_many()` call):

```yaml
# tarantool> res, err = crud.insert('developers', {1, vshard.router.bucket_id({1}), 'A', 'B', 21})
# tarantool> res, {err}
---
- null
- - line: 111
    class_name: InsertError
    err: 'Failed to insert: Duplicate key exists in unique index "primary_index" in
      space "developers" with old tuple - [1, 477, "Alexey", "Adams", 20] and new
      tuple - [1, 477, "A", "B", 21]'
    file: /home/alex/projects/tarantool-meta/crud/crud/insert.lua
    str: 'InsertError: Failed to insert: Duplicate key exists in unique index "primary_index"
      in space "developers" with old tuple - [1, 477, "Alexey", "Adams", 20] and new
      tuple - [1, 477, "A", "B", 21]'
...
```

We can strip it for README:

```yaml
- null
- - class_name: InsertError
    err: 'Failed to insert: Duplicate key exists <...>'
    file: <...>
    str: <...>
```

This way it is okay.

Reply:
So it can be left as it is?

Reply:
I don't insist on any particular option.